sampler_t.adoc

```diff
@@ -46,7 +46,7 @@ The sampler fields are described in the table below:
 
 |
 
-Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized or unnormalized values.
+Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized ([0,1[) or unnormalized ({0, size-1}) values.
```

Comment:
I just tested by accessing the coordinates `{-4.5, -5.0, ..., 15.0, 15.5}` with `CLK_NORMALIZED_COORDS_FALSE` and this was the result:
```
Platforms
        0: Intel(R) OpenCL - OpenCL 1.2
        1: Experimental OpenCL 2.1 CPU Only Platform - OpenCL 2.1
        2: AMD Accelerated Parallel Processing - OpenCL 2.0 AMD-APP (2348.3)
Choose plattform: 2
Devices on plattform AMD Accelerated Parallel Processing
        0: Hawaii - OpenCL 2.0 AMD-APP (2348.3)
        1: Intel(R) Core(TM) i7-5820K CPU @ 3.30GHz - OpenCL 1.2 AMD-APP (2348.3)
Choose device: 0
  coords | CLAMP_TO_EDGE | CLAMP
---------|---------------|---------
-4.50000 |       1.00000 | 0.00000
-3.50000 |       1.00000 | 0.00000
-2.50000 |       1.00000 | 0.00000
-1.50000 |       1.00000 | 0.00000
-0.50000 |       1.00000 | 0.00000
 0.50000 |       1.00000 | 1.00000
 1.50000 |       2.00000 | 2.00000
 2.50000 |       3.00000 | 3.00000
 3.50000 |       4.00000 | 4.00000
 4.50000 |       5.00000 | 5.00000
 5.50000 |       6.00000 | 6.00000
 6.50000 |       7.00000 | 7.00000
 7.50000 |       8.00000 | 8.00000
 8.50000 |       9.00000 | 9.00000
 9.50000 |      10.00000 | 10.00000
10.50000 |      10.00000 | 0.00000
11.50000 |      10.00000 | 0.00000
12.50000 |      10.00000 | 0.00000
13.50000 |      10.00000 | 0.00000
14.50000 |      10.00000 | 0.00000
15.50000 |      10.00000 | 0.00000
```
So, it seems to be that even with unnormalised coordinates it is possible to access every value. This would mean that the actual possible range is `]-\infty,\infty[`, but that is not really helpful in my opinion^^.

I think I would keep `{0, ..., size-1}`, since this is the range of reasonable coordinates. But maybe an additional entry in the notes section describing that it is in both cases possible to use coordinates outside of the defined range?